app/models/good_job/execution.rb

```diff
@@ -206,6 +212,14 @@ def self.queue_parser(string)
 
     # Construct a GoodJob::Execution from an ActiveJob instance.
     def self.build_for_enqueue(active_job, overrides = {})
+      if active_job.priority && !GoodJob.configuration.smaller_number_is_higher_priority
+        ActiveSupport::Deprecation.warn(<<~DEPRECATION)
+          The next major version of GoodJob (v4.0) will change job `priority` to give smaller numbers higher priority, in accordance with Active Job's definition of priority.
+            To opt-in to this behavior now, set `config.good_job.smaller_number_is_higher_priority = true` in your GoodJob initializer or application.rb.
+            To not opt-in yet, but silence this deprecation warning, set `config.good_job.smaller_number_is_higher_priority = false`.
```

Comment:
I think the conditional will always resolve to true when the config value is false, maybe you want to check if it's `nil?`

Reply:
Whoops! yep, that should be a `nil?` check 😓